src/index.js

```diff
@@ -0,0 +1,20 @@
+import './style.css';
+import Task from './modules/tasks';
+
+import { toggle, clearAll } from './modules/list';
+
+const todoItemsList = document.querySelector('#list_navv');
+const clearAllBtn = document.querySelector('#clearAllBTN');
+const task = new Task();
+task.addTask();
+task.render();
+todoItemsList.addEventListener('click', (event) => {
+  if (event.target.type === 'checkbox') {
+    toggle(event.target.parentElement.getAttribute('data-key'), task);
+    task.render();
+  }
```

Comment:
- [ ] Please make sure that the event listener has on the checkbox (`change`) not `click`, so then when we click to the checkbox it will change, the status updates (`completed`: `true` / `false`). :upside_down_face: :upside_down_face: